src/pages/MyProfile.js

```diff
@@ -1,7 +1,64 @@
-const Profile = () => {
-    return (
-        <h1>This is the My Profile Component!</h1>
-    )
+import React from "react"
+import { Link } from "react-router-dom"
+import ProfileDropdown from "../components/ProfileDropdown"
+
+const Profile = ({
+  garden = { firstName: "AM", lastName: "Squad", email: "test@plantie.com" },
+}) => {
+  return (
+    <div className="profile-page">
+      <h2>Hello {garden.firstName}!</h2>
+      <ProfileDropdown
+        dropDownTitle={"Account"}
+        dropDownContent={
+          <div>
+            <p>Name: {garden.firstName + " " + garden.lastName}</p>
+            <p>Email: {garden.email}</p>
+            <Link to="/edit-account">
+              <button>Edit Info</button>
+            </Link>
+          </div>
+        }
+      />
+      <ProfileDropdown
+        dropDownTitle={"Settings"}
+        dropdownContent={
```

Comment:
There are a few inconsistencies in your labeling (might be some in mine too) let's decide if it's dropdown or dropDown and stick with it across the project. this specific instance is labeled differently here and in the ProfileDropdown component. Might cause an issue.

Reply:
Ah! I thought I changed them all to the full camel case dropDownContent. Easy fix.